src/core/Akka/Actor/FSM.cs

```diff
@@ -151,23 +202,39 @@ public abstract class Reason { }
         /// <summary>
         /// Default <see cref="Reason"/> if calling Stop().
         /// </summary>
-        public class Normal : Reason { }
+        public sealed class Normal : Reason
+        {
+            private Normal() { }
```

Comment:
This one could break compatibility. Probably I should keep a public constructor with `ObsoletedAttribute`